Rakefile

```diff
@@ -21,10 +21,6 @@ namespace :test do
 
 end
 
-Rake::Task["test:functional"].enhance do
-  warn "Remember there are still some VMs running, kill them with `vagrant halt` if you are finished using them."
```

Comment:
🗒️ `DockerWrapper` will automatically start Docker Compose on demand and clean it up after tests are done, so this instruction is no longer relevant.